core/src/main/java/com/linecorp/armeria/client/retry/Backoff.java

```diff
@@ -58,6 +58,35 @@ static Backoff exponential(long initialDelayMillis, long maxDelayMillis, double
     }
 
     /**
+     * Returns a {@link Backoff} that computes backoff delay which is a chosen random value between
+     * {@code minDelayMillis} and {@code maxDelayMillis} by {@link ThreadLocalRandom}.
+     */
+    static Backoff random(long minDelayMillis, long maxDelayMillis) {
+        return random(minDelayMillis, maxDelayMillis, ThreadLocalRandom::current);
+    }
+
+    /**
+     * Returns a {@link Backoff} that computes backoff delay which is a random value between
+     * {@code minDelayMillis} and {@code maxDelayMillis}.
+     */
+    static Backoff random(long minDelayMillis, long maxDelayMillis, Supplier<Random> randomSupplier) {
+        return new RandomBackoff(minDelayMillis, maxDelayMillis, ThreadLocalRandom::current);
+    }
+
+    /**
+     * Create a new {@link Backoff} that computes backoff delay using one of
+     * {@link #exponential(long, long, double)}, {@link #fixed(long)} and {@link #random(long, long)} chaining
+     * with {@link #withJitter(double, double)} and {@link #withMaxAttempts(int)}.
+     * For the format of the {@code specification}, please refer to {@link BackoffSpec}.
```

Comment:
You can't refer to `BackoffSpec` here because it's not a public class. Could you put the standalone description here? It would be painful to keep the descriptions in two places, so I recommend just moving it here.